cfme/tests/configure/test_visual_cloud.py

```diff
@@ -100,13 +100,13 @@ def test_cloud_grid_page_per_item(request, page, set_grid, appliance):
         page = page(appliance)
     request.addfinalizer(lambda: go_to_grid(page))
     limit = visual.grid_view_limit
-    navigate_to(page, 'All')
-    tb.select('Grid View')
-    from cfme.web_ui import paginator
-    if paginator.rec_total() is not None:
-        if int(paginator.rec_total()) >= int(limit):
-            assert int(paginator.rec_end()) == int(limit), \
-                "Gridview Failed for page {}!".format(page)
+    view = navigate_to(page, 'All')
+    view.toolbar.view_selector.select('Grid View')
+    min_item, max_item, item_amt = view.paginator.paginator.page_info()
```

Comment:
**Optional:** max_item_amount property can be added here, the same as for view.paginator.items_amount to reduce usage of view.paginator.paginator